lib/network.go

```diff
@@ -462,8 +480,8 @@ func NewTxnMetadata(txType TxnType) (DeSoTxnMetadata, error) {
 		return (&BurnNFTMetadata{}).New(), nil
 	case TxnTypeAuthorizeDerivedKey:
 		return (&AuthorizeDerivedKeyMetadata{}).New(), nil
-	case TxnTypeMessagingGroup:
-		return (&MessagingGroupMetadata{}).New(), nil
+	case TxnTypeAccessGroupCreate:
+		return (&AccessGroupMetadata{}).New(), nil
 	case TxnTypeDAOCoin:
 		return (&DAOCoinMetadata{}).New(), nil
 	case TxnTypeDAOCoinTransfer:
```

Comment:
we need to add the new txn types here as well if we are in fact adding new txn types

Reply:
Yes in the works!